### DISCUSSION TITLE
Undefined array key "errorCode"

### DISCUSSION THREAD

**Discussion by diazsasak** [laravel/framework#40191] [Q&A]
- Laravel Version: 8.77.1
- PHP Version: 8.0.11
- Database Driver & Version: mysql 8.0.26-0ubuntu0.20.04.3
- Meilisearch: 0.24.0
- meilisearch/meilisearch-php: 0.18.3
- laravel/scout: 9.3.4

### Description:

Hi, i am using homestead. i got error when i try to get or create index.

### Steps To Reproduce:

`
$engine = $user->searchableUsing();
return $engine->getOrCreateIndex($user->searchableAs()."_".$user->id);
`

![image](https://user-images.githubusercontent.com/7880137/147554111-2304aed2-49ee-4476-bbc2-f23b4ae65140.png)

![Screenshot 2021-12-28 180010](https://user-images.githubusercontent.com/7880137/147554514-0ee95152-a46a-4d0d-9596-16784cd186d9.png)

i am not sure the bug is at scout code or meilisearch-php code.

thank you for your attention

**Comment by ikramooe** [comment]
any solution to this

**Comment by getda** [comment]
请问解决了吗？
